src/main/java/com/github/algobot76/surabaya/util/FileUnzipper.java

```diff
@@ -0,0 +1,72 @@
+package com.github.algobot76.surabaya.util;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+
+/*
+ * Adapted from https://examples.javacodegeeks.com/core-java/util/zip/zipinputstream/java-unzip-file-example/
+ */
+
+public class FileUnzipper {
+
+	public static void unzip(String filepath, String destpath) {
+		java.io.File directory = new java.io.File(destpath);
+
+		// if the output directory doesn't exist, create it
+		if (!directory.exists())
+			directory.mkdirs();
+
```

Comment:
If `Resource` is used here, we can easily get the input stream using the `getInputStream` method.